### PR TITLE
Incorporate Python-specific feedback on discovery sample gen.

### PR DESCRIPTION
**Pull request by tcoffee-google**
Currently includes only a change in response message printing, from
print(json.dumps(response)) to pprint(response).

### CODE REVIEW

**Comment by vchudnov-g** [comment]
LGTM

File: src/test/java/com/google/api/codegen/testdata/discoveries/py/py_adexchangebuyer.v1.4.json.baseline [hunk 1]
```diff
@@ -10,8 +10,7 @@ PRE-REQUISITES:
    https://cloud.google.com/sdk/ and run 'gcloud beta auth application-default login'
 """
 
-from __future__ import print_function
-import json
+from pprint import pprint
 
 from apiclient import discovery
```
Comment (jonparrott) [inline src/test/java/com/google/api/codegen/testdata/discoveries/py/py_adexchangebuyer.v1.4.json.baseline]: This should be `googleapiclient`, as `apiclient` is a deprecated alias.

**Comment by tcoffee-google** [comment]
@jonparrott PTAL

**Comment by jonparrott** [comment]
LGTM

**Pull request merged**